crates/dojo-world/src/manifest.rs

```diff
@@ -183,6 +185,7 @@ impl From<Manifest<Class>> for Manifest<Contract> {
                 address: None,
                 transaction_hash: None,
                 block_number: None,
+                seed: None,
```

Comment:
We should be able to use `Default::default()` here for a more idiomatic initialization.